SU2_CFD/src/solvers/CDiscAdjMeshSolver.cpp

```diff
@@ -180,9 +171,8 @@ void CDiscAdjMeshSolver::RegisterVariables(CGeometry *geometry, CConfig *config,
 
   /*--- Register reference mesh coordinates ---*/
 
-  for (iPoint = 0; iPoint < nPoint; iPoint++){
-    direct_solver->node[iPoint]->Register_BoundDisp(input);
-  }
+  for (iPoint = 0; iPoint < nPoint; iPoint++)
+    direct_solver->node->Register_BoundDisp(iPoint,input);
```

Comment:
The other failing case disc_adj_flow/mesh_disp_sens relies on the boundary displacement being correctly recorded. I wonder what's the reason for it to fail...